### PR TITLE
Feature portfolio

### PR DESCRIPTION
**Pull request by ibrahim777764**
### MILESTONE 1: setup and mobile version skeleton
understood how to parse a Figma design to create a UI.
how to Flexbox to place elements in the page.
Build a personal portfolio site.

### CODE REVIEW

**Comment by ibrahim777764** [comment]
please review my code, and let me know if there any changes ,thankyou

File: README.md [hunk 1]
```diff
@@ -1 +1,67 @@
...
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (Lordkaito) [inline README.md]: - [ ] [OPTIONAL] It is a good practice to keep your Readme file organized and clean, so please make sure to delete all the unused sections of this file.

File: porthtml.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      <ul id="main-ls">
+        <li><a href="#project-list"></a>Contact</li>
+        <li><a href="#about">Portfolio</a></li>
+        <li><a href="#contact">ABOUT</a></li>
+      </ul>
+    </div>
+  </header>
+  <main class="container1">
+    <div class="con"></div>
+    <section class="sec1">
```
Comment (Lordkaito) [inline porthtml.html:44]: - [ ] Please make sure you follow the design as much as possible and include the background images as in the given template for this section.

File: porthtml.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+
+  <title>ibrahim Portfolio</title>
+ 
+ 
+</head>
+
+<body class="body1">
```
Comment (Lordkaito) [inline porthtml.html:18]: - [ ] Please try to keep all the margins as in the given template. Here, your logo and the hamburger menu are not following the template, as in the given design both of them have some margin which makes them not to be right on the border of the page, so kindly fix this issue. The same thing happens with the section, so please make sure to fix it there too.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @ibrahim777764, 



Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by ibrahim777764** [comment]
please check my code,hope so now there is not reuqirments left,thankyou

File: porthtml.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        <img src="abc.svg" alt="" width="20" height="20">
+
+        <img src="rec.svg" alt="" width="20" height="20">
+
+      </div>
+    </section>
+
+ 
+  
+</main>
```
Comment (gemmen29) [inline porthtml.html]: Please try to match Figma design as match as possible in the background, font size, line height, and social media icons
| Your design | Figma design |
| --- | --- |
| screencapture-raw-githack-ibrahim777764-my-portfoliom-feature-portfolio-porthtml-html-2022-02-11-11_29_46 | Template 2-Mobile-Main |

File: README.md [hunk 5]
```diff
@@ -1 +1,49 @@
...
+build the first 2 sections of the mobile website using the template you chose in the previous step.
+
+<<<<<<< HEAD
+
+![screenshot](./p.png)
+![screenshot](./p1.png)
+
+
+=======
+>>>>>>> 4207069f26825060634c6a9169d4394f5735aec4
```
Comment (gemmen29) [inline README.md]: [OPTIONAL] Kindly try to remove lines `16`, `22` and `23` in your README file as it added when there are conflict in your branch

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @ibrahim777764, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibrahim777764** [comment]
please code review , hope so this time it's all good,thankyou

**Review by alexander16108 — APPROVED** [review]
Hi @ibrahim777764 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**